docs/Advanced-Usage/Debugging/TracerV.rst

```diff
@@ -31,5 +31,25 @@ the "tracing" section.
     startcycle=XXXX
     endcycle=YYYY
 
+TracerV Runtime Arguments (+args)
```

Comment:
Maybe add "midas-level, fpga-level sw simulation only" or something similar to the title here?

Reply:
That's sounds good.